arviz/plots/backends/bokeh/densityplot.py

```diff
@@ -165,35 +185,46 @@ def _d_helper(
         _, hist, edges = histogram(vec, bins=bins)
 
         if outline:
-            ax.quad(
-                top=hist,
-                bottom=0,
-                left=edges[:-1],
-                right=edges[1:],
-                line_color=color,
-                fill_color=None,
-                **extra
+            plotted.append(
+                ax.quad(
+                    top=hist,
+                    bottom=0,
+                    left=edges[:-1],
+                    right=edges[1:],
+                    line_color=color,
+                    fill_color=None,
+                    muted_color=color,
+                    muted_alpha=0.2,
+                    **extra
+                )
             )
         else:
-            ax.quad(
-                top=hist,
-                bottom=0,
-                left=edges[:-1],
-                right=edges[1:],
-                line_color=color,
-                fill_color=color,
-                fill_alpha=shade,
-                **extra
+            plotted.append(
+                ax.quad(
+                    top=hist,
+                    bottom=0,
+                    left=edges[:-1],
+                    right=edges[1:],
+                    line_color=color,
+                    fill_color=color,
+                    fill_alpha=shade,
+                    muted_color=color,
+                    muted_alpha=0.2,
+                    **extra
+                )
             )
 
     if hpd_markers:
-        ax.diamond(xmin, 0, line_color="black", fill_color=color, size=markersize)
-        ax.diamond(xmax, 0, line_color="black", fill_color=color, size=markersize)
+        plotted.append(ax.diamond(xmin, 0, line_color="black", fill_color=color, size=markersize))
+        plotted.append(ax.diamond(xmax, 0, line_color="black", fill_color=color, size=markersize))
 
     if point_estimate is not None:
         est = calculate_point_estimate(point_estimate, vec, bw)
-        ax.circle(est, 0, fill_color=color, line_color="black", size=markersize)
+        plotted.append(ax.circle(est, 0, fill_color=color, line_color="black", size=markersize))
 
     _title = Title()
     _title.text = vname
     ax.title = _title
+    ax.title.text_font_size = "13pt"
+
+    return plotted
```

Comment:
Why is plotted returned? Shouldn't the axes array be returned?

Reply:
`plotted` is the list of glyphs that are needed to specify legend.
We are first plotting the axes in main function, and then adding plots (`glyph renders`) on top of it, using list of glyphs from helper function. So, the helper function needs only to get the list of glyphs, not axes.

Reply:
Ooh, I see now, sorry